website/api/hooks/custom/index.js

```diff
@@ -153,7 +153,13 @@ will be disabled and/or hidden in the UI.
 
             // No session? Proceed as usual.
             // (e.g. request for a static asset)
-            if (!req.session) { return next(); }
+            // If the request is for production assets in the `/dist` folder, we'll set headers to disable caching.
+            if (!req.session) {
```

Comment:
i.e. instead of this change, cutting this:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/618009/167000926-f6285e3a-9c49-4f8f-92b1-2d3f7cd95e22.png">

and moving it up top above this `if`

Reply:
Also re this comment:
```
// If the request is for production assets in the `/dist` folder, we'll set headers to disable caching.
```

The problem is not that the assets are being cached.  The problem is that the HTML itself is being cached, thus referencing old bundles

Reply:
I added a commit with this.